charts/rancher-webhook/values.yaml

```diff
@@ -12,4 +12,10 @@ capi:
   enabled: false
 
 mcm:
-  enabled: true
\ No newline at end of file
+  enabled: true
+
+preDelete:
+  enabled: true
+  image:
+    repository: rancher/kubectl
```

Comment:
Did we confirm that this is the correct minimal image to use? Looks correct, but just wanted to double check.

Reply:
Yep, the same image and tag are used in multiple latest charts in rancher/charts dev-v2.6, such as [monitoring 100.0.0+up16.6.0](https://github.com/rancher/charts/blob/abcc294eed065e8dec10a1757ed9dd5f808d15f4/charts/rancher-monitoring/rancher-monitoring/100.0.0%2Bup16.6.0/values.yaml#L521-L524) 
and [fleet-agent 100.0.0+up0.3.6-rc10](https://github.com/rancher/charts/blob/abcc294eed065e8dec10a1757ed9dd5f808d15f4/charts/fleet-agent/fleet-agent/100.0.0%2Bup0.3.6-rc10/values.yaml#L55-L57)